src/main/fc/rc_adjustments.c

```diff
@@ -652,6 +679,13 @@ static uint8_t applySelectAdjustment(adjustmentFunction_e adjustmentFunction, ui
         }
 #endif
         break;
+    case ADJUSTMENT_OSD_PROFILE:
+        if (getOSDprofile() != position) {
+            setOSDprofile(position);
+            beeps = position + 1;
```

Comment:
I don't think this is something worth beeping for when you get obvious visual feedback already.

Reply:
Yes and no...  There are beeps for other profile changes, also it makes it clear which profile is selected, handy for configuration.  May also be good to have an OSD element for this, similar to the one for rate profiles, maybe something that is only displayed for a second or so when the profile is changed.

Reply:
> There are beeps for other profile changes

That is the problem, it creates ambiguity. You will immediately know if the setting is correct or not based on that is shown in the OSD so having beeps (which you may not even hear at range) is pointless.